ambry-tools/src/main/java/com.github.ambry/tools/admin/BlobValidator.java

```diff
@@ -368,38 +373,38 @@ private void validateBlobOnAllReplicas(BlobId blobId, ClusterMap clusterMap, boo
       }
     }
     compareListOfBlobContent(blobId.getID(), replicaIdBlobContentMap);
-    logger.info("Summary ");
+    boolean errorLoggingRequired = false;
+    if (responseMap.keySet().size() != 1 || !replicaIdBlobContentMap.keySet().contains(ServerErrorCode.No_Error)) {
+      errorLoggingRequired = true;
+    }
+    if (errorLoggingRequired) {
+      logger.error("Summary of responses for " + blobId);
+    } else {
+      logger.debug("Summary of responses for " + blobId);
+    }
     for (ServerErrorCode serverErrorCode : responseMap.keySet()) {
-      logger.info(serverErrorCode + ": " + responseMap.get(serverErrorCode));
+      if (errorLoggingRequired) {
+        logger.error(serverErrorCode + ": " + responseMap.get(serverErrorCode));
```

Comment:
"ErrorLogging" is not what user sent as an arg to the tool. This variable is set if the expected response from every server is not in sync with each other or if not for No_Error. If everything is good, nothing is logged.